R/plotRegion.R

```diff
@@ -354,7 +373,7 @@ plotRegion <- function(
                      referenceCoordinate = referenceCoordinate),
                      tr[!names(tr) %in% c("trackData", "trackType", "grl",
                                           "region", "referenceCoordinate",
-                                          "labelAccuracy")]
+                                          "labelAccuracy", "fillColors")]
```

Comment:
It was not totally clear to me why `fillColors` was added here (same for line 385 below)